components/more-less/demo/more-less.html

```diff
@@ -29,8 +29,8 @@ <h2>More-less with height</h2>
 
 			<d2l-demo-snippet>
 				<template>
-					<d2l-more-less height="6em">
-						<p id="img-target">Hi everybody! <a href="">Hi!</a></p>
+					<d2l-more-less height="8em">
+						<p id="img-target">Hi everybody!Hi everybody!Hi everybody!Hi everybody!Hi everybody!Hi everybody! <a href="">Hi!</a></p>
```

Comment:
Nit: It might look cleaner to replace this with a "Lorem ipsum" sentence to be consistent with the other demos